spec/index.html

```diff
@@ -538,72 +538,70 @@ <h2>RDF Graphs</h2>
   <section id="section-triples">
     <h3>Triples</h3>
 
-    <p>An <dfn data-local-lt="triple">RDF triple</dfn> is a 3-tuple that is defined recursively as follows:</p>
+    <p>An <dfn data-local-lt="triple">RDF triple</dfn> (usually called "triple")
+      is a 3-tuple (|s|, |p|, |o|) where:</p>
+
+    <ul>
+      <li>|s| is an <a>IRI</a> or a <a>blank node</a>,</li>
+      <li>|p| is an <a>IRI</a>, and</li>
+      <li>|o| is an <a>IRI</a>, a <a>blank node</a>, a <a>literal</a>,
+        or a <a>triple term</a>.</li>
+    </ul>
+
+    <p>A <dfn>triple term</dfn> is a 3-tuple that is defined recursively as follows:</p>
 
     <ul>
       <li>
         If |s| is an <a>IRI</a> or a <a>blank node</a>,
         |p| is an <a>IRI</a>, and
         |o| is an <a>IRI</a>, a <a>blank node</a>, or a <a>literal</a>,
-        then (|s|, |p|, |o|) is an RDF triple.
+        then (|s|, |p|, |o|) is a triple term.
```

Comment:
I agree that having 'triple term' here, instead of 'RDF triple', deviates from the grammar in the seeking-consensus table. Yet, if we are aiming to clearly distinguish between the concept of an RDF triple (as something that can be used as an element of RDF graphs) and the concept of a triple term (as something that can (recursively) be used as a component of RDF triples), then it should be 'triple term' here.